Makefile

```diff
@@ -13,6 +13,10 @@ build:
 	@echo "  >  Building binaries..."
 	@CGO_ENABLED=0 go build -o ${PROJECTNAME} ./cmd/...
 
+build-arm:
+	@echo "  >  Building binaries..."
+	@CGO_ENABLED=0 env GOOS=linux GOARCH=arm64 go build -o ${PROJECTNAME} ./cmd/...
```

Comment:
As @artek-koltun indicated to unify with godpu and you can also have GOOS=$(uname) to default to the local OS.  If a different OS is needed such as cross-compile, then

GOOS ?= $(shell go env GOOS) # detect the os target from the environment variables

Reply:
@atulpatel261194 will take care of this

Reply:
done